hashtables/ex2/ex2.js

```diff
@@ -6,8 +6,37 @@ class Ticket {
 }
 
 const reconstruct_trip = (tickets, length) => {
-    //your code here
-    
+    let hash_tickets = {}
+    let route = []
+
+    tickets.forEach(ticket => {
+        hash_tickets[ticket.source] = ticket.destination
+    })
+
+    for(let i = 0; i < length; i++){
```

Comment:
A different way of doing this would be to push the None right off the bat
```js
route.push(flights['NONE'])
    
    for(let i = 0; i < length-1; i++){
        route.push(hash_tickets[route[i]])
    }
return route;
```